text-to-speech/TTS_inference_API_notebook.ipynb

```diff
@@ -0,0 +1,383 @@
+        "import nest_asyncio\n",
+        "from pyngrok import ngrok\n",
+        "import base64\n",
+        "import tempfile\n",
+        "\n",
+        "nest_asyncio.apply()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# run <! tts --list_models> to see available models\n",
+        "model=\"tts_models/en/ljspeech/glow-tts\" #download model of your choice"
+      ],
+      "metadata": {
+        "id": "TEWy55CuTsC8"
+      },
+      "execution_count": 3,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def tts_call(text: str, model: str=model):\n",
+        "  # Create a temporary file to save the output audio\n",
+        "  with tempfile.NamedTemporaryFile(suffix=\".wav\", delete=False) as temp_file:\n",
+        "    out_path = temp_file.name\n",
+        "\n",
+        "  # Run the TTS command\n",
+        "  subprocess.run([\"tts\", \"--model_name\", model, \"--text\", text, \"--out_path\", out_path])\n",
+        "\n",
+        "  # Return the path to the saved audio file\n",
+        "  return out_path"
+      ],
+      "metadata": {
+        "id": "9hP6rwEbN1uz"
+      },
+      "execution_count": 4,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "app = FastAPI()\n",
+        "app.add_middleware(\n",
+        "    CORSMiddleware,\n",
+        "    allow_origins=['*'],\n",
+        "    allow_credentials=True,\n",
+        "    allow_methods=['*'],\n",
+        "    allow_headers=['*'],\n",
+        ")"
+      ],
+      "metadata": {
+        "id": "XOQw6YSsNV5e"
+      },
+      "execution_count": 8,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class TextRequest(BaseModel):\n",
+        "    text: str\n",
+        "\n",
+        "class OutputText(BaseModel):\n",
+        "  answer: str"
+      ],
+      "metadata": {
+        "id": "okc52MET8GZp"
+      },
+      "execution_count": 9,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "@app.get(\"/synthesize_audios\")\n",
+        "def synthesize_audio(payload: TextRequest):\n",
+        "    # Call your TTS function to generate the audio file\n",
+        "    print(\"text:\", payload.text)\n",
+        "    audio_file_path = tts_call(payload.text)\n",
+        "\n",
+        "    # Return the audio file as the response\n",
+        "    return FileResponse(audio_file_path, media_type=\"audio/wav\")\n",
+        "\n",
+        "@app.get(\"/\", response_model=OutputText)\n",
+        "async def root():\n",
+        "    return {\"answer\": \"Hello from TTS\"}"
+      ],
+      "metadata": {
+        "id": "xbHL2ZCNNYu3"
+      },
+      "execution_count": 10,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!ngrok config add-authtoken 2S28VLMNCGx4zukfhelsfj4jPHd_oErC1e1zpvJKZkpecfPZ #<place your access token here>"
```

Comment:
remove the auth token you have here as well.

Reply:
let's use similar approach as https://github.com/nsmq-ai/nsmqai/pull/51/files#diff-d755e2bb4a53a4cf5f1dcfb59c460f3cbc400da5cdb40a512d867038c1efc11cR195 and https://github.com/nsmq-ai/nsmqai/pull/51/files#diff-d755e2bb4a53a4cf5f1dcfb59c460f3cbc400da5cdb40a512d867038c1efc11cR194